src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs

```diff
@@ -42,5 +43,24 @@ public async Task<IActionResult> GetList()
                 return BadRequest();
             }
         }
+
+        [HttpGet]
+        [Route("{standardUId}")]
+        public async Task<IActionResult> GetByStandardUId(string standardUId)
+        {
+            try
+            {
+                var queryResponse = await _mediator.Send(new GetStandardDetailsQuery(standardUId));
+
+                if (queryResponse.StandardDetails == null) return NotFound();
+
+                return Ok((GetStandardDetailsResponse)queryResponse.StandardDetails);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, $"Error attempting to get standard using {standardUId}");
+                return BadRequest();
```

Comment:
Bad Request or Internal Server Error?

Reply:
ISE for sure... but has to be done at a later stage to avoid conflicts.